layers/+lang/typescript/packages.el

```diff
@@ -84,49 +84,49 @@
     (spacemacs/add-to-hooks #'smartparens-mode '(typescript-mode-hook
                                                  typescript-tsx-mode-hook))))
 
-(defun typescript/init-tide ()
-  (use-package tide
-    :defer t
-    :commands (typescript/jump-to-type-def)
-    :config
-    (progn
-      (spacemacs/declare-prefix-for-mode 'typescript-mode "mE" "errors")
-      (spacemacs/declare-prefix-for-mode 'typescript-tsx-mode "mE" "errors")
-      (spacemacs/declare-prefix-for-mode 'typescript-mode "mg" "goto")
-      (spacemacs/declare-prefix-for-mode 'typescript-tsx-mode "mg" "goto")
-      (spacemacs/declare-prefix-for-mode 'typescript-mode "mh" "help")
-      (spacemacs/declare-prefix-for-mode 'typescript-tsx-mode "mh" "help")
-      (spacemacs/declare-prefix-for-mode 'typescript-mode "mn" "name")
-      (spacemacs/declare-prefix-for-mode 'typescript-tsx-mode "mn" "name")
-      (spacemacs/declare-prefix-for-mode 'typescript-mode "mr" "refactor")
-      (spacemacs/declare-prefix-for-mode 'typescript-tsx-mode "mr" "refactor")
-      (spacemacs/declare-prefix-for-mode 'typescript-mode "mS" "server")
-      (spacemacs/declare-prefix-for-mode 'typescript-tsx-mode "mS" "server")
-      (spacemacs/declare-prefix-for-mode 'typescript-mode "ms" "send")
-      (spacemacs/declare-prefix-for-mode 'typescript-tsx-mode "ms" "send")
-
-      (setq keybindingList '("Ee" tide-fix
-                             "Ed" tide-add-tslint-disable-next-line
-                             "gb" tide-jump-back
-                             "gg" tide-jump-to-definition
-                             "gt" spacemacs/typescript-jump-to-type-def
-                             "gu" tide-references
-                             "hh" tide-documentation-at-point
-                             "ri" tide-organize-imports
-                             "rr" tide-rename-symbol
-                             "rf" tide-rename-file
-                             "sr" tide-restart-server)
-            typescriptList (cons 'typescript-mode keybindingList)
-            typescriptTsxList (cons 'typescript-tsx-mode
-                                    (cons "gg" (cons 'tide-jump-to-definition
-                                                     keybindingList ))))
-      (apply 'spacemacs/set-leader-keys-for-major-mode typescriptList)
-      (apply 'spacemacs/set-leader-keys-for-major-mode typescriptTsxList)))
-
-  (add-to-list 'spacemacs-jump-handlers-typescript-tsx-mode
-               '(tide-jump-to-definition :async t))
-  (add-to-list 'spacemacs-jump-handlers-typescript-mode
-               '(tide-jump-to-definition :async t)))
+;; (defun typescript/init-tide ()
```

Comment:
Please remove this code, it is obsolete now. If someone wants to see the history he can still check with git.